spec/javascripts/rich_text_module_spec.js

```diff
@@ -134,5 +147,26 @@ describe("RichTextModule", function() {
 
     });
 
+  it("does not alter new line spaced lists", function() {
+
+    module.setMode('wysiwyg')
+
+    module.value('<p>1. first<br/>2. second<br/>3. third</p><p>1.</p><p>2.</p><p>3.</p>')
+
+    module.setMode('markdown')
+
+    expect(module.html()).toContain('<p>1. first<br>2. second<br>3. third</p><p>1.</p><p>2.</p><p>3.</p>')
+    
+    module.setMode('markdown')
+    
+    module.value('1. one\n2. two\n3. three\n\n1.\n\n2.\n\n3.')
```

Comment:
Take a look here - i entered this as in the string, convert to wysiwyg and back to markdown, and hope to find the same string there. But I kind of expect I won't... let's see!